app/models/concerns/variant_stock.rb

```diff
@@ -130,8 +130,9 @@ def save_stock
   end
 
   def raise_error_if_no_stock_item_available
-    message = 'You need to save the variant to create a stock item before you can set stock levels.'
-    raise message if stock_items.empty?
+    # puts self.id
```

Comment:
this comment needs to be removed

Reply:
what is comment harming there?

Reply:
Imo this commented puts statment is just adding noise here. Its a debug statment.